docs/self-managed/identity/orchestration-identity/configuration.md

```diff
@@ -20,38 +20,38 @@ The default configuration available below is for Camunda 8 Run. Helm installatio
-| `SPRING_PROFILES_ACTIVE`                            | **Note:** This property will be deprecated as additional authentication methods become available. | `consolidated-auth`           |
+| `CAMUNDA_PERSISTENT_SESSIONS_ENABLED`               | Enables shared authentication between the Orchestration web applications (Operate and Tasklist).  | `true`                        |
 | `CAMUNDA_SECURITY_AUTHENTICATION_METHOD`            | The authentication method to use.                                                                 | `basic`                       |
 | `CAMUNDA_SECURITY_AUTHENTICATION_UNPROTECTEDAPI`    | If the API can be used without authentication.                                                    | `true`                        |
-| `CAMUNDA_PERSISTENT_SESSIONS_ENABLED`               | Enables shared authentication between the Orchestration web applications (Operate and Tasklist).  | `true`                        |
 | `CAMUNDA_SECURITY_AUTHORIZATIONS_ENABLED`           | If authorizations are enabled.                                                                    | `true`                        |
-| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_USERNAME` | The username of the first user.                                                                   | `demo`                        |
-| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_PASSWORD` | The password of the first user.                                                                   | `demo`                        |
-| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_NAME`     | The name of the first user.                                                                       | Demo                          |
 | `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_EMAIL`    | The email address of the first user.                                                              | `demo@demo.com`               |
+| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_NAME`     | The name of the first user.                                                                       | Demo                          |
+| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_PASSWORD` | The password of the first user.                                                                   | `demo`                        |
+| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_USERNAME` | The username of the first user.                                                                   | `demo`                        |
 | `CAMUNDA_SECURITY_MULTITENANCY_ENABLED`             | Enables multi-tenancy. This requires the api to be protected.                                     | `false`                       |
+| `SPRING_PROFILES_ACTIVE`                            | **Note:** This property will be deprecated as additional authentication methods become available. | `consolidated-auth`           |
```

Comment:
@conceptualshark what do you think of sorting this in alphabetical order?

Reply:
The other tab of this chart is the Helm properties, where I'm not sure if it's helpful to have them in the same order they're in the Helm chart (though I am also not sure if this order reflects that, either!). 

If that's not needed, then I am good with reordering this, as long as both tabs are in the same order.

Reply:
It doesn't seem like it was in any specific order. Since helm properties that belong together will start in the same way they'll always be grouped together here in alphabetical order as well.